plugins/CoreHome/javascripts/uiControl.js

```diff
@@ -67,6 +75,9 @@
                     throw new Error("Error: " + control.constructor.name + "'s destroy method does not call " +
                                     "UIControl.destroy. You may have a memory leak.");
                 }
+            } else {
+                // Control is still active / used.
+                activeControls.push(control);
             }
         }
```

Comment:
Isn't here something missing like `UIControl._controls = activeControls;`, in order to keep track of those controls not removed by the current run of `cleanupUnusedControls `